libraries/ESP8266WiFi/src/include/UdpContext.h

```diff
@@ -441,24 +462,13 @@ class UdpContext
             const ip_addr_t *srcaddr, u16_t srcport)
     {
         (void) upcb;
-        // check receive pbuf chain depth
-        {
-            pbuf* p;
-            int count = 0;
-            for (p = _rx_buf; p && ++count < rxBufMaxDepth*2; p = p->next);
-            if (p)
-            {
-                // pbuf chain too deep, dropping
-                pbuf_free(pb);
-                DEBUGV(":udr\r\n");
-                return;
-            }
-        }
 
```

Comment:
did you try to increase `rxBufMaxDepth` instead of removing that block ?

Reply:
No, might also help, but what would be the right value?
What is the reason to omit UDP messages? Memory usage?

Reply:
The 'submodules' problem should exist here too, right?

Reply:
Redid the same process as with the WiFiServer change

Reply:
Yes, memory usage